File: java-maven/files/src/test/java/{{ .PackagePath }}/{{ .NameWord }}/Performance.java
package {{ .Package }}.{{ .NameWord }};

import io.gatling.javaapi.http.HttpProtocolBuilder;

import static io.gatling.javaapi.http.HttpDsl.http;
import static org.galaxio.gatling.javaapi.SimulationConfig.baseUrl;

public final class Performance {

    public static final HttpProtocolBuilder httpProtocol = http
            .baseUrl(baseUrl())
            .acceptHeader("application/json")
            .contentTypeHeader("application/json")
            .disableFollowRedirect();

    private Performance() {
    }
}

File: kotlin-maven/files/src/test/kotlin/{{ .PackagePath }}/{{ .NameWord }}/Performance.kt
package {{ .Package }}.{{ .NameWord }}

import io.gatling.javaapi.http.HttpDsl.http
import io.gatling.javaapi.http.HttpProtocolBuilder
import org.galaxio.gatling.javaapi.SimulationConfig.baseUrl

object Performance {
    val httpProtocol: HttpProtocolBuilder = http
        .baseUrl(baseUrl())
        .acceptHeader("application/json")
        .contentTypeHeader("application/json")
        .disableFollowRedirect()
}
